api/v1/models_rhelv2_test.go

```diff
@@ -277,3 +278,68 @@ func TestComponentsFromDatabaseModelRHELv2(t *testing.T) {
 	assert.Equal(t, expectedRHELv2PkgEnvs, imgComponents.RHELv2PkgEnvs)
 	assert.ElementsMatch(t, expectedNotes, imgComponents.Notes)
 }
+
+func Test_getFullRHELv2Features(t *testing.T) {
```

Comment:
can we just add a test to also ensure we skip it for non-rhcos?

Reply:
Oh, I saw your comment post-merge. I will follow up on the UT in a separate PR.